etc/config.sample.toml

```diff
@@ -95,6 +95,35 @@ reporting-disabled = false
   # log any sensitive data contained within a query.
   # query-log-enabled = true
 
+  # Settings for the TSM engine
+
+  # CacheMaxMemorySize is the maximum size a shard's cache can
```

Comment:
Not right now. We want it to be global eventually, but remember that it was wonky because the cache isn't in the engine?